Airports/EGPK/SMR/Regions.txt

```diff
@@ -547,6 +547,19 @@ terminal N055.30.16.282 W004.35.43.804
          N055.30.15.329 W004.35.42.451
          N055.30.15.846 W004.35.42.118
 
+;Hanger Perpendicular to H
```

Comment:
```suggestion
;Hangar Perpendicular to H
```

Reply:
Nah, I just added new coordinates in, that's why if I was to create a kmz and then convert it to sct and then remake the smr completely with new regions and updated files, I probably would have made the display list to eliminate the green outline of the grey polygon, the apron. It just wouldn't be worth the hassle in my opinion.